lib/deep_cover/load.rb

```diff
@@ -15,9 +15,9 @@ def load_absolute_basics
       require_relative 'config_setter'
       require_relative 'tools/camelize'
       AUTOLOAD.each do |module_name|
-        DeepCover.autoload(Tools::Camelize.camelize(module_name), "#{__dir__}/#{module_name}")
+        DeepCover.autoload(Tools::Camelize.camelize(module_name), File.join(__dir__, module_name.to_s))
```

Comment:
I thought this was working ok even on Windows. If so, I'd keep the nicer version and make sure we are compatible... Otherwise let me know coz I use `'/'` all the time for `require` and such

Reply:
Done